resources/gdc_resources/data_dictionary/administrative/center.json

```diff
@@ -0,0 +1,76 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "additionalProperties": false,
+    "category": "administrative",
+    "description": "A Professional Organization or Group which has or is able to submit data to the GDC that are valid with respect to GDC prescribed or defined data formats. (GDC subclass of NCIt C19711)",
+    "downloadable": false,
+    "id": "center",
+    "links": [],
+    "namespace": "https://gdc.cancer.gov",
+    "previous_version_downloadable": false,
+    "program": "*",
+    "project": "*",
+    "properties": {
+        "center_type": {
+            "description": "Type classification of the center (e.g. CGCC).",
+            "type": "string"
+        },
+        "code": {
+            "description": "Numeric code for the center.",
+            "type": "string"
+        },
+        "id": {
+            "common": {
+                "description": "A 128-bit identifier. Depending on the mechanism used to generate it, it is either guaranteed to be different from all other UUIDs/GUIDs generated until 3400 AD or extremely likely to be different. Its relatively small size lends itself well to sorting, ordering, and hashing of all sorts, storing in databases, simple allocation, and ease of programming in general.",
+                "termDef": {
+                    "cde_id": "C54100",
+                    "cde_version": null,
+                    "source": "NCIt",
+                    "term": "Universally Unique Identifier",
+                    "term_url": "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&version=16.02d&ns=NCI_Thesaurus&code=C54100"
+                }
+            },
+            "description": "UUID for the center.",
+            "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+            "systemAlias": "node_id",
+            "type": "string"
+        },
+        "name": {
+            "description": "Name of the center (e.g. Broad Institute of MIT and Harvard).",
+            "type": "string"
+        },
+        "namespace": {
+            "description": "Domain name of the center (e.g. broad.mit.edu).",
+            "type": "string"
+        },
+        "short_name": {
+            "description": "Shortened name of the center (e.g. BI).",
+            "type": "string"
+        },
+        "type": {
+            "type": "string"
+        }
+    },
+    "required": [
+        "code",
+        "namespace",
+        "name",
+        "short_name",
+        "center_type"
```

Comment:
does this need to be required? could be use cases where new data doesn't have a clear center type (ie. category of center). does seem that short_name (name of center) could hold info similar to this. aka would it make sense to move center_type to not be required as long as short_name is?

Reply:
great point! will definitely have that in mind. For now all required elements are defined via GDC or FHIR. In this case, the data dictionary in resources is pulling GDC dictionary down to keep track of the data version locally as I go through the mappings (mapping list with source -> destination dictionaries).